### ISSUE TITLE
Add ability to replicate pan behavior as seen in native keyboard view.

### ISSUE THREAD

**Issue by illyabusigin** [illyabusigin/CYRKeyboardButton#2]
I'd like to implement a behavior found on the iPhone's system keyboard:  user touches a key, and other keys are highlighted as the finger drags across the keyboard.  When the finger lifts, the last highlighted key inserts the text.  If user long-taps a key, then its expandedInputView pan gesture is tracked instead of the entire keyboard.

**Comment by cemkozinoglu** [comment]
+1 this would be awesome

**Comment by khantthulinn** [comment]
Yes. How can we do that one? I am modifying this library and still not okay. Can anyone advice?
